### PR TITLE
PHOENIX-5401 Error in running apache.phoenix.mapreduce.index.automati…

### PR DESCRIPTION
**Pull request by kadirozde**
…on.PhoenixMRJobSubmitter

### CODE REVIEW

File: phoenix-core/src/main/java/org/apache/phoenix/mapreduce/PhoenixServerBuildIndexInputFormat.java [hunk 1]
```diff
@@ -99,9 +98,6 @@ protected  QueryPlan getQueryPlan(final JobContext context, final Configuration
             if (txnScnValue != null) {
                 scan.setAttribute(BaseScannerRegionObserver.TX_SCN, Bytes.toBytes(Long.valueOf(txnScnValue)));
             }
-
-            // Initialize the query plan so it sets up the parallel scans
-            queryPlan.iterator(MapReduceParallelScanGrouper.getInstance());
```
Comment (priyankporwal) [inline phoenix-core/src/main/java/org/apache/phoenix/mapreduce/PhoenixServerBuildIndexInputFormat.java]: Why is this removed @kadirozde?

Comment (kadirozde) [reply phoenix-core/src/main/java/org/apache/phoenix/mapreduce/PhoenixServerBuildIndexInputFormat.java]: The same step is done again after the calling this method so it is not necessary to do here. Please see the super class where this method is called. You can also looked at the default implementation of this method which does not this step and the other step that I removed above this.

**Comment by swaroopak** [comment]
Thank you @kadirozde  for the patch. LGTM, +1
